website/docs/r/monitor_scheduled_query_rules_alert.html.markdown

```diff
@@ -0,0 +1,153 @@
+---
+subcategory: "Monitor"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_monitor_scheduled_query_rules_alert"
+description: |-
+  Manages an AlertingAction Scheduled Query Rule within Azure Monitor
+---
+
+# azurerm_monitor_scheduled_query_rules_alert
+
+Manages an AlertingAction Scheduled Query Rule within Azure Monitor.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "monitoring-resources"
+  location = "West US"
+}
+
+resource "azurerm_application_insights" "example" {
+  name                = "appinsights"
+  location            = var.location
+  resource_group_name = azurerm_resource_group.example.name
+  application_type    = "web"
+}
+
+resource "azurerm_log_analytics_workspace" "example" {
+  name                = "loganalytics"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku                 = "PerGB2018"
+  retention_in_days   = 30
+}
+
+# Example: Alerting Action
+resource "azurerm_scheduled_query_rule_alert" "example" {
+  name                = format("%s-queryrule", var.prefix)
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  action {
+    action_group           = []
+    email_subject          = "Email Header"
+    custom_webhook_payload = "{}"
+  }
+  data_source_id = azurerm_application_insights.example.id
+  description    = "Scheduled query rule Alerting Action example"
+  enabled        = true
+  frequency      = 5
+  query          = "requests | where status_code >= 500 | summarize AggregatedValue = count() by bin(timestamp, 5m)"
+  query_type     = "ResultCount"
+  severity       = 1
+  time_window    = 30
+  trigger {
+    threshold_operator = "GreaterThan"
```

Comment:
```suggestion
    operator = "GreaterThan"
```

Reply:
When using this resource I get an error that `threshold_operator` is not supported and that I'm missing the `operator` argument

Reply:
Thank you!  I pushed an update.